Translations update from Kodi Weblate

weblate opened the pull request:
Translations update from [Kodi Weblate](https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/)
for Kodi add-ons: skins/skin.estuary.



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widgets/kodi-add-ons-skins/-/skin-estuary/horizontal-auto.svg)

AlwinEsch commented on addons/skin.estuary/language/resource.language.ar_sa/strings.po:
```diff
@@ -27,7 +27,7 @@ msgstr "Estuary هو المظهر الافتراضي ل كودي الإصدار
 msgctxt "Addon Disclaimer"
 msgid "Estuary is the default skin for Kodi, removing it may cause issues"
-msgstr ""
+msgstr "‫Estuary هو المظهر الافتراضي ل كودي, إزالته قد تسبب بعض المشاكل"
```
Think here was gone something wrong and maybe request to close?

gade01 replied:
This a new warning in Github.
The translation includes a right to left embedding character: `U+202B`.
A lot of our translations include this character.
I tested some of them and didn't find any issues, but I don't know the exact issues it might lead to.

For this string, I removed the character by copying some of the translation from another string (with the exact same words).

weblate committed:
Translated using Weblate (Arabic (Saudi Arabia) (ar_sa))

Currently translated at 91.4% (150 of 164 strings)

Translated using Weblate (Persian (Iran) (fa_ir))

Currently translated at 75.2% (73 of 97 strings)

Translated using Weblate (Persian (Iran) (fa_ir))

Currently translated at 85.9% (141 of 164 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (97 of 97 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (164 of 164 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (5 of 5 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (French (France) (fr_fr))

Currently translated at 100.0% (5 of 5 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Estonian (et_ee))

Currently translated at 81.4% (79 of 97 strings)

Translated using Weblate (Estonian (et_ee))

Currently translated at 99.3% (163 of 164 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Estonian (et_ee))

Currently translated at 76.2% (74 of 97 strings)

Translated using Weblate (Estonian (et_ee))

Currently translated at 96.3% (158 of 164 strings)

Translated using Weblate (Italian (it_it))

Currently translated at 100.0% (5 of 5 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Swedish (sv_se))

Currently translated at 80.0% (4 of 5 strings)

Translated using Weblate (Swedish (sv_se))

Currently translated at 80.0% (4 of 5 strings)

Translated using Weblate (Lithuanian (lt_lt))

Currently translated at 97.9% (95 of 97 strings)

Translated using Weblate (Italian (it_it))

Currently translated at 97.9% (95 of 97 strings)

Translated using Weblate (Lithuanian (lt_lt))

Currently translated at 100.0% (164 of 164 strings)

Translated using Weblate (Italian (it_it))

Currently translated at 100.0% (164 of 164 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Spanish (Mexico) (es_mx))

Currently translated at 100.0% (97 of 97 strings)

Translated using Weblate (Spanish (Mexico) (es_mx))

Currently translated at 100.0% (164 of 164 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (97 of 97 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (164 of 164 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 92.7% (90 of 97 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 94.5% (155 of 164 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Indonesian (id_id))

Currently translated at 92.0% (151 of 164 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Turkish (tr_tr))

Currently translated at 100.0% (97 of 97 strings)

Translated using Weblate (Turkish (tr_tr))

Currently translated at 98.7% (162 of 164 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Danish (da_dk))

Currently translated at 100.0% (97 of 97 strings)

Translated using Weblate (Danish (da_dk))

Currently translated at 100.0% (164 of 164 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Korean (ko_kr))

Currently translated at 100.0% (97 of 97 strings)

Translated using Weblate (Korean (ko_kr))

Currently translated at 100.0% (164 of 164 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Chinese (China) (zh_cn))

Currently translated at 100.0% (97 of 97 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (97 of 97 strings)

Translated using Weblate (Chinese (China) (zh_cn))

Currently translated at 100.0% (164 of 164 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (164 of 164 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (German (de_de))

Currently translated at 100.0% (97 of 97 strings)

Translated using Weblate (German (de_de))

Currently translated at 100.0% (164 of 164 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Polish (pl_pl))

Currently translated at 100.0% (97 of 97 strings)

Translated using Weblate (Polish (pl_pl))

Currently translated at 100.0% (164 of 164 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Estonian (et_ee))

Currently translated at 100.0% (152 of 152 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Estonian (et_ee))

Currently translated at 100.0% (152 of 152 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Estonian (et_ee))

Currently translated at 100.0% (152 of 152 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Arabic (Saudi Arabia) (ar_sa))

Currently translated at 100.0% (152 of 152 strings)

Co-authored-by: Alfonso Cachero <alfonso.cachero@gmail.com>
Co-authored-by: Christian Gade <gade@kodi.tv>
Co-authored-by: Edson Armando <edsonarmando78@outlook.com>
Co-authored-by: Emir Sarı <bitigchi@me.com>
Co-authored-by: Fajar Maulana <jeremy.belpois.einstein@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kai Sommerfeld <kai.sommerfeld@gmx.com>
Co-authored-by: Luna Jernberg <droidbittin@gmail.com>
Co-authored-by: Marco Spezzamonte <marco@spezzamonte.it>
Co-authored-by: Marek Adamski <fevbew@wp.pl>
Co-authored-by: Minho Park <parkmino@gmail.com>
Co-authored-by: Oskari Lavinto <olavinto@protonmail.com>
Co-authored-by: Raimondas Dužinskas <raimondas.duzinskas@gmail.com>
Co-authored-by: Venice Man <marco@spezzamonte.it>
Co-authored-by: rimasx <riks_12@hot.ee>
Co-authored-by: taxigps <taxigps@sina.com>
Co-authored-by: vdkbsd <valexgus@gmail.com>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/da_dk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/de_de/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/es_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/es_mx/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/et_ee/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/fa_ir/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/fi_fi/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/it_it/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/ko_kr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/lt_lt/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/pl_pl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/ru_ru/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/tr_tr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/zh_cn/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/ar_sa/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/da_dk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/de_de/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/es_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/es_mx/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/et_ee/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/fa_ir/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/fi_fi/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/id_id/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/it_it/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/ko_kr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/lt_lt/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/pl_pl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/ru_ru/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/tr_tr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/zh_cn/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/audioencoder-kodi-builtin-aac/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/audioencoder-kodi-builtin-aac/fi_fi/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/audioencoder-kodi-builtin-aac/fr_fr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/audioencoder-kodi-builtin-aac/sv_se/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/audioencoder-kodi-builtin-wma/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/audioencoder-kodi-builtin-wma/fi_fi/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/audioencoder-kodi-builtin-wma/sv_se/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/screensaver-xbmc-builtin-dim/fi_fi/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/screensaver-xbmc-builtin-dim/it_it/
Translation: Kodi add-ons: skins/skin.estouchy
Translation: Kodi add-ons: skins/skin.estuary
Translation: Kodi core/audioencoder.kodi.builtin.aac
Translation: Kodi core/audioencoder.kodi.builtin.wma
Translation: Kodi core/screensaver.xbmc.builtin.dim
